services/gateway/shared/registerRpcApi.js

```diff
@@ -26,15 +26,22 @@ const { validate } = require('./paramValidator');
 
 const apiMeta = [];
 
-const configureApi = (apiName, apiPrefix) => {
+const configureApi = (apiNames, apiPrefix) => {
+	const allMethods = {};
 	const transformPath = url => {
 		const dropSlash = str => str.replace(/^\//, '');
 		const curlyBracketsToColon = str => str.split('{').join(':').replace(/}/g, '');
 
 		return curlyBracketsToColon(dropSlash(url));
 	};
-
-	const allMethods = Utils.requireAllJs(path.resolve(__dirname, `../apis/${apiName}/methods`));
+	if (typeof apiNames === 'string') apiNames = [apiNames];
+	apiNames.map(apiName => {
+		Object.assign(
+			allMethods,
+			Utils.requireAllJs(path.resolve(__dirname, `../apis/${apiName}/methods`)),
+		);
+		return apiName;
+	});
```

Comment:
Perhaps we could simplify it to:

```suggestion
	apiNames.forEach(apiName => Object.assign(
		allMethods,
		Utils.requireAllJs(path.resolve(__dirname, `../apis/${apiName}/methods`)),
	));
```

Reply:
Done!